kratos/solving_strategies/strategies/butcher_tableau.h

```diff
@@ -94,6 +87,8 @@ class ButcherTableau
     ///@name Life Cycle
     ///@{
 
+    /// Destructor.
+    virtual ~ButcherTableau() = default;
```

Comment:
```suggestion
```

let the compiler do its job ;)

No need for this to be virtual unless you derive from it

Reply:
Agree. However a got a warning in a branch I was working on because of this :confused:

Reply:
really?
what is the warning?

Reply:
If I don't remember wrong `class has virtual functions but non-virtual destructor`.

Reply:
Ah yes that makes sense

Then I would remove the virtual on the functions

Reply:
We're deriving so we need to leave it as it is.